<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paivakirja</class>
 <widget class="QWidget" name="Paivakirja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Alkupäivä</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="alkupvm"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Loppupäivä</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="loppupvm"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="kohdennusCheck">
       <property name="text">
        <string>Kohdennuksella</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="kohdennusCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="jarjestysRyhma">
     <property name="title">
      <string>Tulostusjärjestys</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="aikajarjestysRadio">
        <property name="text">
         <string>Päivämäärä</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tositejarjestysRadio">
        <property name="text">
         <string>Tositenumero</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tulostakohdennuksetCheck">
     <property name="text">
      <string>Tulosta kohdennukset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ryhmittelelajeittainCheck">
     <property name="text">
      <string>Ryhmittele tositelajeittain</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tulostasummat">
     <property name="text">
      <string>Tulosta summarivit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kohdennusCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>kohdennusCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
